Failures in "CoreCLR Common Pri1 Test Build AnyOS AnyCPU" pipeline.

I have rebased my changes on the top of the current master and triggered jitstress and jitstressregs, both have failed with:
```
/Users/runner/runners/2.169.1/work/1/s/eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/ [/Users/runner/runners/2.169.1/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj]
##[error]eng/liveBuilds.targets(147,5): error : The 'libs' subset must be built before building this project. Missing artifacts: /Users/runner/runners/2.169.1/work/1/s/artifacts/bin/ref/microsoft.netcore.app/Release/

```
in `CoreCLR Common Pri1 Test Build AnyOS AnyCPU checked` that I have not seen before.
[example](https://dev.azure.com/dnceng/public/_build/results?buildId=672645&view=logs&j=db768a9e-3325-5ce3-2d52-896c40bee68a&t=ea573249-0a9e-59d9-0a41-40ec8869f91f)

I think it could be caused by #37373, @safern could you take a look?

We need stress testing to be green to push two PRs that have been blocked on testing for a few days due to infrastructure issues. It would be very helpful to get this fixed/reverted before new potential disrupting infra changes.

cc @jashook @dotnet/runtime-infrastructure @CarolEidt

--- a/eng/pipelines/coreclr/gc-simulator.yml
+++ b/eng/pipelines/coreclr/gc-simulator.yml
@@ -39,6 +39,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: gc-simulator
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/gcstress-extra.yml
+++ b/eng/pipelines/coreclr/gcstress-extra.yml
@@ -34,6 +34,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: gcstress-extra
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/gcstress0x3-gcstress0xc.yml
+++ b/eng/pipelines/coreclr/gcstress0x3-gcstress0xc.yml
@@ -34,6 +34,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: gcstress0x3-gcstress0xc
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jit-experimental.yml
+++ b/eng/pipelines/coreclr/jit-experimental.yml
@@ -35,6 +35,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: jit-experimental
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jitstress-isas-arm.yml
+++ b/eng/pipelines/coreclr/jitstress-isas-arm.yml
@@ -35,6 +35,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: jitstress-isas-arm
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jitstress.yml
+++ b/eng/pipelines/coreclr/jitstress.yml
@@ -42,6 +42,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: jitstress
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jitstress2-jitstressregs.yml
+++ b/eng/pipelines/coreclr/jitstress2-jitstressregs.yml
@@ -42,6 +42,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: checked
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jitstressregs-x86.yml
+++ b/eng/pipelines/coreclr/jitstressregs-x86.yml
@@ -36,6 +36,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: jitstressregs-x86
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/coreclr/jitstressregs.yml
+++ b/eng/pipelines/coreclr/jitstressregs.yml
@@ -42,6 +42,7 @@ jobs:
     - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
     jobParameters:
       testGroup: jitstressregs
+      liveLibrariesBuildConfig: Release
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:

--- a/eng/pipelines/libraries/base-job.yml
+++ b/eng/pipelines/libraries/base-job.yml
@@ -49,9 +49,6 @@ jobs:
         - _crossBuildPropertyArg: ''
         - _testRunNamePrefixSuffix: ''
 
-        # we need to override this value to support build-coreclr-and-libraries-job.yml
-        - _BuildConfig: ${{ parameters.buildConfig }}
-
         - librariesBuildArtifactName: ${{ format('libraries_bin_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
 
         - ${{ if ne(parameters.testScope, '') }}:
@@ -111,6 +108,9 @@ jobs:
 
         - _buildArguments: $(_runtimeConfigurationArg) -configuration ${{ parameters.buildConfig }} -ci -arch ${{ parameters.archType }} $(_finalFrameworkArg) $(_testScopeArg) $(_runtimeOSArg) $(_msbuildCommonParameters) $(_runtimeArtifactsPathArg) $(_crossBuildPropertyArg)
         - ${{ parameters.variables }}
+
+        # we need to override this value to support build-coreclr-and-libraries-job.yml
+        - _BuildConfig: ${{ parameters.buildConfig }}
 
       dependsOn:
       - checkout

--- a/eng/pipelines/libraries/base-job.yml
+++ b/eng/pipelines/libraries/base-job.yml
@@ -49,6 +49,9 @@ jobs:
         - _crossBuildPropertyArg: ''
         - _testRunNamePrefixSuffix: ''
 
+        # we need to override this value to support build-coreclr-and-libraries-job.yml
+        - _BuildConfig: ${{ parameters.buildConfig }}
+
         - librariesBuildArtifactName: ${{ format('libraries_bin_{0}{1}_{2}_{3}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
 
         - ${{ if ne(parameters.testScope, '') }}:

